src/web/mjs/engine/ChaptermarkManager.mjs

```diff
@@ -35,7 +35,8 @@ export default class ChaptermarkManager extends EventTarget {
      * Will reset chaptermarks when saving fails.
      */
     _syncChaptermarks( callback ) {
-        Engine.Storage.saveConfig( 'chaptermarks', this.chaptermarks, 2 )
+        //HYPOFLEX
+        Engine.Storage.saveBookmarks( 'chaptermarks', this.chaptermarks, 2 )
```

Comment:
Will chaptermarks be saved after changing the directory in settings?
If not they may be lost when the directory is changed and the application is closed right after.

If not it might be a good idea to save the chaptermarks whenever the settings changed event is fired
Example: https://github.com/manga-download/hakuneko/blob/618ecde29a464e6a838fe03fb6c04738317f11d7/src/web/mjs/connectors/NineAnime.mjs#L46